src/addon/file_server/directory_entry.rs

```diff
@@ -62,4 +65,6 @@ pub struct DirectoryIndex {
     pub(crate) breadcrumbs: Vec<BreadcrumbItem>,
     pub(crate) sort_by_name: bool,
     pub(crate) sort_by_size: bool,
+    pub(crate) sort_by_date_created: bool,
+    pub(crate) sort_by_date_modified: bool,
```

Comment:
Given that sorting can only have one value at a time, I think perhaps is better to have an enum here?

```rust
enum Sort {
  Name,
  Size,
  DateCreated,
  DateModified,
}
```

Then we can use that value to sort down the line

Reply:
That would make sense